main/acle.md

```diff
@@ -351,6 +351,14 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
 * Added a requirement on [`arm_new_za`](#arm_new_za) to set the initial
   contents of ZA to zero.
 
+#### Changes for next release
```

Comment:
Do you mean Changes in _this_ release?

Reply:
I think the convention is to use “Changes in the next release” for changes that been made to the document since the last release (in other words, for changes that haven't yet been released, but that are queued up for the next/upcoming release).  The “next release” part is then replaced with a release version just before the release.

Reply:
That's correct. That piece of text will be replaced in the release process.